public/config/config.staging

```diff
@@ -2,7 +2,7 @@ export default ({
   tfApiUrl: 'https://radar.threefold.io/api/',
   jsApiUrl: `${window.location}actors/`,
 
-  apiUrl: 'https://ffc-signaling.staging.jimber.org',
+  apiUrl: '${window.location}/freeflowconnect-signaling',
```

Comment:
@azertyalex Why is this?

Reply:
@azertyalex This would give a double slash I think

Reply:
i need a ` not a '.
dammit